docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+
+  consul-agent-1: &consul-agent
+    image: consul:latest
+    networks:
+      - trento
+    command: "agent -retry-join consul-server-bootstrap -client 0.0.0.0"
+
+  consul-agent-2:
+    <<: *consul-agent
+
+  consul-agent-3:
+    <<: *consul-agent
+
+  consul-server-1: &consul-server
+    <<: *consul-agent
+    command: "agent -server -retry-join consul-server-bootstrap -client 0.0.0.0"
+
+  consul-server-2:
+    <<: *consul-server
+
+  consul-server-bootstrap:
+    <<: *consul-agent
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+      - "8600:8600/udp"
+    command: "agent -server -bootstrap-expect 3 -ui -client 0.0.0.0"
```

Comment:
To be honest, I find the usage of yaml anchors a little confusing. How would you feel about making everything a little more explicit and repeating the commands instead?

Also, personal nitpick, I would rename `consul-server-bootstrap` to just `consul-server-1`, but that's just me. 😛

Reply:
I personally don't have a strong preference here. I'd usually lean towards repeating less, but tbh the anchors here are indeed not very easy to read specially if one is not familiar with it. I'll change it.